packages/react-components/react-theme/src/themes/ThemeTypography.stories.tsx

```diff
@@ -71,3 +71,47 @@ export const FontWeight = () => (
     )}
   </div>
 );
+
+export const TypographyStyles = () => {
+  function formatTypographyStyleValue(typographyStyleValue: TypographyStyle) {
```

Comment:
nit: Add some comments about what formatting is happening instead of relying on the understanding of regex lookups